js/main.js

```diff
@@ -13,17 +13,22 @@ $form.addEventListener('submit', subButton);
 
 function subButton(event) {
   const $eform = document.querySelector('.eform');
+  const $li = document.querySelectorAll('li');
   event.preventDefault();
   let entry = {};
-
   if (data.editing !== null) {
-
     for (let i = 0; i < data.entries.length; i++) {
       // eslint-disable-next-line eqeqeq
```

Comment:
Don't add eslint rules without asking me! If you're getting an error, it's probably for a reason